fxmanifest.lua

```diff
@@ -2,18 +2,13 @@ fx_version 'cerulean'
 game 'gta5'
 
 description 'QB-Core'
-version '1.1.0'
+version '1.2.0'
 
 shared_scripts {
 	'shared/locale.lua',
 	'locale/en.lua', -- replace with desired language
 	'config.lua',
-	'shared/main.lua',
-	'shared/items.lua',
-	'shared/jobs.lua',
-	'shared/vehicles.lua',
-	'shared/gangs.lua',
-	'shared/weapons.lua'
+	'shared/*.lua'
```

Comment:
please add shared/main.lua first and then globbing (just due to the fact that main is QBShared = {} the rest are QBShared = QBShared or {} )

Reply:
If you look at shared/main.lua I changed it to `QBShared = QBShared or {}` so it can load like the rest, the functions don't depend on each other in the shared so the loading order doesn't matter that way